caluma/caluma_workflow/managers.py

```diff
@@ -0,0 +1,92 @@
+from django.db.models import Manager
+
+from caluma.caluma_core.events import send_event
+from caluma.caluma_user.models import BaseUser
+
+from ..caluma_form.models import Document
+from . import events, models, utils
+
+
+class CaseManager(Manager):
+    def _validate(self, data):
```

Comment:
Django has infrastructure for model-level validation: https://docs.djangoproject.com/en/3.0/ref/models/instances/#django.db.models.Model.clean_fields . 
I suggest using this instead of using a custom method